modules/apps/foundation/frontend-theme/justin-theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,129 @@
+                  ${site_name} </span>
+                 <p class="hidden-xs">Test Page for Liferay Theme by Justin Kim</p>
+            </#if>
+
+          <button class="navbar-toggle collapsed align-bottom" id="btnCollapse" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+              <span id="btnChild">MENU</span>
+          </button>
+         
+        </div>
+
+          <#if !is_signed_in>
+            <a data-redirect="${is_login_redirect_required?string}" href="${sign_in_url}" id="sign-in" rel="nofollow">${sign_in_text}</a>
+          </#if>
+
+          <#if has_navigation && is_setup_complete>
+            <#include "${full_templates_path}/navigation.ftl" />
+          </#if>
+
+        </div>
+      </div>
+  </header>
+
+  <section id="content">
+      <h1 class="hide-accessible">${the_title}</h1>
+
+      <#if selectable>
+        <@liferay_util["include"] page=content_include />
+      <#else>
+        ${portletDisplay.recycle()}
+
+        ${portletDisplay.setTitle(the_title)}
+
+        <@liferay_theme["wrap-portlet"] page="portlet.ftl">
+          <@liferay_util["include"] page=content_include />
+        </@>
+      </#if>
+
+
+
+  </section>
+
+    <footer id="footer" role="contentinfo">
+      <div class="row">
+        <p class="powered-by col-xs-6">
+          <@liferay.language key="powered-by" /> <a href="http://www.liferay.com" rel="external">Liferay</a>
+        </p>
+
+        <div class="social-media-wrapper col-xs-6">
+          
+          <ul>
+              <li>
+                  <div id="facebook"><a class="icon-facebook"
+                  href="http://www.facebook.com/pages/Liferay/45119213107" 
+                  target="_blank"><span class="hide">Facebook</span></a></div>
+              </li>
+              <li>
+                  <div id="twitter"><a class="icon-twitter" 
+                  href="http://www.twitter.com/liferay" 
+                  target="_blank"><span class="hide">Twitter</span></a></div>
+              </li>
+              <li>
+                  <div id="linked-in"><a class="icon-linkedin"
+                  href="http://www.linkedin.com/company/83609" 
+                  target="_blank"><span class="hide">LinkedIn</span></a></div>
+              </li>
+              <li>
+                  <div id="youtube"><a class="icon-youtube"
+                  href="http://www.youtube.com/user/liferayinc" 
+                  target="_blank"><span class="hide">YouTube</span></a></div>
+              </li>
+              <li>
+                  <div id="google-plus">
+                  <a class="icon-google-plus"
+                  href="https://plus.google.com/+liferay/posts" 
+                  target="_blank"><span class="hide">Google</span></a></div>
+              </li>
```

Comment:
We should look into making these configurable by the end user through theme settings.